CONTRIBUTING.md

```diff
@@ -8,6 +8,7 @@ implementation and other Rust Bitcoin-related projects, which are hosted in the
 mostly guidelines, not rules. Use your best judgment, and feel free to propose
 changes to this document in a pull request.
 
+
```

Comment:
My initial idea was to use two lines only before 2-level headings and not the rest, to help visually maintain large-level structure

Reply:
No worries, I did that for you and did the removal of @sgeisler from maintainer file based on his email.